R/compiled.R

```diff
@@ -109,5 +144,42 @@ line_coverages <- function(source_file, matches, values, functions) {
   names(res) <- lapply(res, function(x) key(x$srcref))
 
   class(res) <- "line_coverages"
+
+  res
+}
+
+br_coverages <- function(source_file, matches, values, functions) {
+
+  src_file <- srcfilecopy(source_file, readLines(source_file))
+
+  line_lengths <- vapply(src_file$lines[as.numeric(matches$line)], nchar, numeric(1))
+
+  res <- Map(function (br, line, length, value, func) {
+    src_ref <- srcref(src_file, c(line, 1, line, length))
+    res <- list(srcref = src_ref, value = value, functions = func, parent = NULL, pos = 1L, default = as.logical(br))
+    class(res) <- "br_coverage"
+    res},
+    matches$branch, matches$line, line_lengths, values, functions)
+
+  if (!length(res)) {
+    return(NULL)
+  }
+
+  names(res) <- lapply(res, function(x) key(x$srcref))
+
+  class(res) <- "br_coverages"
+
   res
 }
+
+fill <- function(m) {
```

Comment:
I would call it fill_line and move it inside the only function that calls it - with a comment explaining what does it do

Reply:
Done!